hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterDataSerializerHook.java

```diff
@@ -47,31 +91,150 @@ public int getFactoryId() {
 
     @Override
     public DataSerializableFactory createFactory() {
-        return FACTORY;
-    }
+        ConstructorFunction<Integer, IdentifiedDataSerializable>[] constructors = new ConstructorFunction[LEN];
 
```

Comment:
All non-Operations have been already here in the Hook, see the changes below. I've only added operations,.